### ISSUE TITLE
Bug with trying to open characters with a space in the name or the folder structure from Windows Explorer

### ISSUE THREAD

**Issue by TomSmeagol** [richardwilkes/gcs#139]
Pretty much what the title says. If the character name uses a space and/or the character is in a folder with a space in it, GURPS Character Sheet displays an error when trying to open the character from Windows Explorer.

**Comment by richardwilkes** [comment]
I assume you can just use the File > Open menu option to open the file without issue, though, right?

It wouldn't surprise me if the new installer tooling did the wrong thing when generating the necessary integration bits for Windows. Unfortunately, it is an early pre-release. I'm using it solely because the Java folks removed what I used to use back in Java 9... and still haven't replaced it yet (in theory, Java 14... which is where this came from -- but they've been saying the "next release" since they took it away).

Anyway, bottom line is that it probably won't get fixed until the Java folks make their actual release of the packager -- and hopefully fix that issue in the process.

**Comment by TomSmeagol** [comment]
Sure, it still opens from the program, but since shell integration is a fundamental aspect of Windows applications, I thought it was worth reporting. I also thought that, since it does work if there isn’t a space in the file or path name, that it might be a fairly simple bug to track down and fix. Sounds like that is not the case, though. Thanks for the response!

--Tom

On Dec 8, 2019, at 11:34 AM, Richard Wilkes <notifications@github.com> wrote:

﻿

I assume you can just use the File > Open menu option to open the file without issue, though, right?

It wouldn't surprise me if the new installer tooling did the wrong thing when generating the necessary integration bits for Windows. Unfortunately, it is an early pre-release. I'm using it solely because the Java folks removed what I used to use back in Java 9... and still haven't replaced it yet (in theory, Java 14... which is where this came from -- but they've been saying the "next release" since they took it away).

Anyway, bottom line is that it probably won't get fixed until the Java folks make their actual release of the packager -- and hopefully fix that issue in the process.

**Comment by richardwilkes** [comment]
I agree -- pretty basic. I'm very disappointed that this fails. Thanks for reporting it.

**Comment by TomSmeagol** [comment]
BTW for the record I am so grateful for your program—I love GURPS and my gaming group and I are always making new characters and we love that there is such a great tool out there to aid in character creation. Thanks for all of your hard work!

--Tom

On Dec 8, 2019, at 11:48 AM, Richard Wilkes <notifications@github.com> wrote:

﻿

I agree -- pretty basic. I'm very disappointed that this fails. Thanks for reporting it.
